data/Nigeria/Senate/sources/instructions.json

```diff
@@ -21,6 +21,21 @@
         "from": "group-wikidata",
         "source": "manual/group_wikidata.csv"
       }
-    }
+    },
+    {
+      "file": "morph/shineyoureye.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "struan/nigeria-shineyoureye",
+        "query": "SELECT *, NULL as position FROM data WHERE position = 'Senator'"
```

Comment:
I don't think I understand deeply enough why this is being included here (or indeed what the two different values might be used for) and what the lifespan of the usefulness is. 

Possibly the most visible usage will be that once this is in the data, EveryPolitician.org will start displaying it on each card, and I'd like to be able to turn that into a link to the ShineYourEye site. If you're not maintaining URL-compatibility between the two versions of the site, then we'll need to tweak the identifier → URL mapping at that point, but that's fairly simple to do. If the prior identifier also becomes largely meaningless at that point, then I'm not sure it'll be worth continuing to include it.

I suspect you're also wanting to include it/them, though, so that it makes something else easier at your end?